docs/reference/commandline/container_run.md

```diff
@@ -326,7 +326,32 @@ are broken into multiple containers, you might need to share the IPC mechanisms
 of the containers, using `"shareable"` mode for the main (i.e. "donor")
 container, and `"container:<donor-name-or-ID>"` for other containers.
 
-### <a name="privileged"></a> Full container capabilities (--privileged)
+### <a name="privileged"></a> Escalate container privileges (--privileged)
+
+The `--privileged` flag gives the following capabilities to a container:
```

Comment:
Also wondering if we should add a (more detailed?) section in the https://docs.docker.com/go/daemon-access/ section.

Reply:
(asking because it's not immediately obvious to me what `--privileged` has to do with post-install actions, or with user group permissions)

Reply:
Ah, sorry, more in line of the "daemon attack vector". Being able to start a privileged container is a prime example of "bad things" people could do when having access to the API (hope that makes sense)